src/integrator_interface.jl

```diff
@@ -9,6 +9,9 @@ function savevalues!(integrator::DDEIntegrator, force_save=false)
     integrator.integrator.k = integrator.k
     integrator.integrator.t = integrator.t
 
+    # update solution of DDE integrator
+    invoke(savevalues!, Tuple{AbstractDDEIntegrator,Bool}, integrator, force_save)
```

Comment:
isn't this pointing to the same solution as the ODEIntegrator? Why does it need its own save?